### PR TITLE
security: modify configuration names in SE and TLS

### PR DESCRIPTION
**Pull request by pillip8282**
Previous configuration names for SE are artik
specific(ex. TLS_WITH_SSS...) As TizenRT would support various SE,
these names can confuse developers. And some name doesn't look proper
like SUPPORT_FULL_SECURITY which set Secure Storage support.
Furthermore TLS and SE share configurations so it shows same options
both SE and TLS menu. But TLS doesn't want to set H/W accelerator even
though SE support it. For example if the SE couldn't support all key
length algorithms, then software logic in TLS should be used.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d4c2c185e320dffee92d7bc32a520cdf7cb76f4d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d4c2c185e320dffee92d7bc32a520cdf7cb76f4d] - Code Rule Check Result:
framework/src/st_things/things_stack/security/sss_security/sss_storage.c:36: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)

File: framework/src/st_things/things_stack/security/sss_security/sss_storage.c [hunk 1]
```diff
@@ -89,42 +88,69 @@ size_t sss_write_mem(const void *ptr, size_t size, size_t count, FILE *stream)
...
+	if (res != SECURITY_OK) {
+		free(hnd);
+		return NULL;
+	}
 
-	if (ret != 0) {
-		printf("setSecureStorageType sss_sync fail : %d\n", ret);
-		return ret;
+	security_data input = {sss_fs.data, sss_fs.size};
+	//security_data input = {g_ss_pool, g_ss_pool_size};
```
Comment (NilabhShant) [inline framework/src/st_things/things_stack/security/sss_security/sss_storage.c]: Please remove this line if not needed.

**Comment by seinfra** [comment]
Target : [3fed5b115c5eb34465e6b7de348eb362b178c5d7] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3fed5b115c5eb34465e6b7de348eb362b178c5d7] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6ca1c0746e5bed1139f742845766eced5c2bb391] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6ca1c0746e5bed1139f742845766eced5c2bb391] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [87864f9175667e3507eb8f0f41279923cf5e140f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [87864f9175667e3507eb8f0f41279923cf5e140f] - Code Rule Check OK.

**Pull request merged**